src/panels/logbook/ha-logbook-renderer.ts

```diff
@@ -560,6 +560,7 @@ class HaLogbookRenderer extends LitElement {
         .icon-message {
           display: flex;
           align-items: center;
+          column-gap: 8px;
```

Comment:
We shouldn't use gaps without using grid

Reply:
But this is a flex, gaps for flex too.

Reply:
Thats okay that it works. I would rather use the margin as mentioned below to keep things consistent in the code

Reply:
Agree, using a margin is a possible solution.
But some things are easier to solve on a higher level - i.e. on the flexbox container level (.icon-message).
Assume that in future something will change - like "3rd column is added", then we will face this "tight problem" again.

Reply:
You will need to update margin-inline-end as well

Reply:
Not sure if it is needed since the element is never rotated...
But OK - let's set `margin-inline-end` as well!